e2e/credentials-server.e2e.spec.ts

```diff
@@ -238,8 +239,10 @@ describeWithCreds('Credentials server', () => {
 
     await doRequestExpectStatus('erase', bobsURL, 200);
 
-    stdout = await doRequest('get', bobsURL);
-    expect(stdout).toContain('credentials not found in native keychain');
+    if (credStore !== 'pass') {
```

Comment:
Please add a comment on why `pass` is being singled out. (I realize that's in the PR description, but that's not in code…)

You also have the option of using `test.skip(credStore === 'pass', '<explanation here>')` instead. Though that may be misleading since it marks the whole thing as skipped — might need a new test (or, if possible, a subtest) instead.

Reply:
I'd rather keep each test standalone. The comment after this block, saying why we don't test erasing a non-existent serverURL, makes it harder to write a single skippable test that tests `erase && get`